clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/view/model/KubernetesV2ServerGroup.java

```diff
@@ -231,6 +232,13 @@ public LoadBalancerServerGroup toLoadBalancerServerGroup() {
         .build();
   }
 
+  @Deprecated
+  @Null
+  @Override
+  public ImageSummary getImageSummary() {
```

Comment:
Why do we need this?

Reply:
The interface [requires this](https://github.com/spinnaker/clouddriver/blob/924d53a28ebbbb3391ed1fc93ca44a897ec4d092/clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/ServerGroup.java#L170) even though it's deprecated in favor of the similarly-named `getImagesSummary()`.

Prior to this change we had a never-initialized field `imageSummary` whose auto-generated getter provided this.  But as we were never setting the field, this seemed clearer than changing the field to `private final ImageSummary imageSummary = null` (which would have been equivalent).